code/game/gamemodes/management/event/combat.dm

```diff
@@ -65,6 +65,12 @@ GLOBAL_VAR_INIT(wcorp_enemy_faction, "") //decides which faction WCorp will be u
 				addtimer(CALLBACK(src, PROC_REF(endround)), 90 MINUTES)
 				to_chat(world, span_userdanger("This week will last 90 minutes."))
 
+			//Fixers
+			if("railway")
+				addtimer(CALLBACK(src, PROC_REF(roundendwarning)), 15 MINUTES)
+				addtimer(CALLBACK(src, PROC_REF(loseround)), 20 MINUTES)
+				to_chat(world, span_userdanger("This week will last 90 minutes."))
+
```

Comment:
update to upstream/master and lets use the new funky automatic lose thingy shall we?
```suggestion
				RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, PROC_REF(CheckLiving))
```